ui/src/localStorage.js

```diff
@@ -2,9 +2,22 @@ export const loadLocalStorage = () => {
   try {
     const serializedState = localStorage.getItem('state')
 
-    return JSON.parse(serializedState) || {}
+    const state = JSON.parse(serializedState)
+
+    // eslint-disable-next-line no-undef
+    if (state.VERSION !== VERSION) {
+      // eslint-disable-next-line no-console
+      console.log('New version detected. Clearing old settings.')
```

Comment:
Could we either warn pre-emptively or notify via UI? I'm concerned specifically about a user who doesn't think to look in the `console` not realizing that they're going to lose their `queryConfigs` in Data Explorer, or not realizing that that's why it happened.